tests/sentry/replays/test_organization_selector_index.py

```diff
@@ -95,7 +95,7 @@ def test_get_replays(self):
             assert response_data["data"][0]["project_id"] == project.id
             assert (
                 response_data["data"][0]["dom_element"]
-                == 'div#myid.class1.class2[role="button"][alt="Alt"][testid="1"][aria="AriaLabel"][title="MyTitle"]'
+                == 'div#myid.class1.class2[role="button"][alt="Alt"][testid="1"][aria="AriaLabel"][title="MyTitle"][component_name="SignUpForm"]'
```

Comment:
Should this be `data-sentry-component` in line with the test suite?

Reply:
I was using `testid` for reference and noticed that `testid` was used for elements and clicks and `data-test-id` was used for attributes. So the element, dom_element and clicks, I used `component_name` but for searches/attributes, I used `data-sentry-component`. Do we want a different behaviour?

Reply:
Seems fine then if its just a stylistic choice.  For what its worth, I think `testid` should be renamed to match the search filter some day.  Or the search filter needs to be updated to target `testid` instead of `data-test-id`.

Reply:
hmm I can put up another PR later to try changing the search filter to target `testid` and `component_name`